packages/console/src/pages/SignInExperience/components/SignInMethodsForm.tsx

```diff
@@ -49,10 +52,19 @@ const SignInMethodsForm = () => {
               disabled={primaryMethod === method}
               {...register(`signInMethods.${method}`)}
             />
+            {method === SignInMethodKey.Email && email && (
+              <ConnectorSetupWarning type={ConnectorType.Email} />
+            )}
+            {method === SignInMethodKey.SMS && sms && (
+              <ConnectorSetupWarning type={ConnectorType.SMS} />
+            )}
+            {method === SignInMethodKey.Social && social && (
+              <ConnectorSetupWarning type={ConnectorType.Social} />
+            )}
```

Comment:
should this switch logic implement within the ConnectorSetupWarning itself?

Reply:
makes sense

Reply:
updated